doc/source/appendix/ontologies.rst

```diff
@@ -38,30 +39,34 @@ What is the minimum attribute requirement for  OntologyTerm in GA4GH?
 
 Conceptually (and consistent with the metadata branch)
 
-:ontologyId:
-  required and implemented as URI
-  we assume this resolves to a meaningful document, e.g. http://purl.obolibrary.org/obo/SO_0000147
+:termId:
+  The unique identifier of the term in an ontology source. This should be a CURIE (e.g., SO:0001234), but may be an alphanumerical string (e.g., 8000/3) if no CURIE are available.
 :term:
   preferred but not required (e.g. ‘exon’); corresponds to class label
+:termUri:
+  A fully qualified URI that can be used to access details regarding the term. e.g. http://purl.bioontology.org/ontology/SNOMEDCT/76345009.
```

Comment:
I think we need to think about having the URI further - we have been discussing having a prefix registry (see https://github.com/ga4gh/schemas/issues/695) so this may be up for change/removal.